tests/executor/test_strategy.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from yapapi.executor.strategy import LeastExpensiveLinearPayuMS
+
+from tests.factories.rest.market import OfferProposalFactory
+
+
```

Comment:
I'd consider replacing this whole test suite with a single test similar to this one:
```
@pytest.mark.asyncio
async def test_LeastExpensiveLinearPauyuMS_score():

    from itertools import product

    expected_time = 60
    cpu_prices = [0.0, 0.01, 0.03, 0.1, 0.3, 1.0, 3.0]
    time_prices = [0.0, 0.01, 0.03, 0.1, 0.3, 1.0, 3.0]
    fixed_prices = [0.0, 0.01, 0.03, 0.1, 0.3, 1.0, 3.0]

    triples = product(cpu_prices, time_prices, fixed_prices)
    triples = [(coeffs[0] * 1.01, coeffs[1] * 1.001, coeffs[2] * 1.0001) for coeffs in triples]

    strategy = LeastExpensiveLinearPayuMS(expected_time_secs=expected_time)

    def cost(coeffs):
        return coeffs[0] * expected_time + coeffs[1] * expected_time + coeffs[2]

    scores = {}
    for coeffs in triples:
        offer = OfferProposalFactory(**{"proposal__proposal__properties__linear_coeffs": coeffs})
        score = await strategy.score_offer(offer)
        scores[coeffs] = score

    # Sort coeffs by cost, from the most expensive to the least expensive
    triples_by_cost = sorted(triples, key=cost, reverse=True)
    # Sort coeffs by stragey score, from the lowest (worst) to the highest (best)
    triples_by_score = sorted(triples, key=lambda coeffs: scores[coeffs])
    assert triples_by_cost == triples_by_score
```

Reply:
hmm... I still think the few separate tests that I proposed are more readable and obvious than this ... let's discuss it on a call then